test/workflows/components/workflows.libsonnet

```diff
@@ -238,8 +259,13 @@
                     dependencies: ["setup-cluster", "build"],
                   },
                   {
-                    name: "run-tests",
-                    template: "run-tests",
+                    name: "simple-tfjob-cpu",
```

Comment:
Take a look at:
https://github.com/kubeflow/kubeflow/blob/master/testing/workflows/components/workflows.libsonnet#L101

One thing we can do is auto generate the step graph from a list of dependencies. If you look at that file we just define tasks which is a list of dictionaries. We then autogenerate from that the various Argo templates.

Reply:
I added a TODO for this. After this PR the boiler plate stuff should be reduced to manageable size, and adding new test cases (instead of test suites) should not require editing the jsonnet files anymore.